### DISCUSSION TITLE
Fan Start and Stop Speed Records?

### DISCUSSION THREAD

**Discussion by SimonZerafa** [Rem0o/FanControl.Releases#1761] [Q&A]
HI,

During the Fan Discovery process the stop and start speed percentages are discovered and presumably recorded somewhere?

Where would this be recorded and perhaps this information could be part of each fan's information card in future?

There is little point setting a fan curve to extend below the minimum stop percentage as they fan will already be stopped (or not in the case of some PWM fans which never stop completely).

KInd Regards

Simon

**Comment by Rem0o** [comment]
![image](https://user-images.githubusercontent.com/14118956/236710046-c27237ff-a536-4a24-8a4a-2713e8f20a9b.png)
